assets/javascripts/discourse/templates/admin/plugins-salesforce.hbs

```diff
@@ -0,0 +1 @@
+<a href="https://login.salesforce.com/services/oauth2/authorize?client_id={{siteSettings.salesforce_client_id}}&redirect_uri={{Discourse.BaseUrl}}&response_type=token">Authorize</a>
```

Comment:
IIRC the `Discourse.BaseUrl` global is deprecated... is this printing a deprecation notice?